src/test/scala/uk/gov/hmrc/test/ui/pages/section5/CommodityDetailsPage.scala

```diff
@@ -49,4 +52,15 @@ object CommodityDetailsPage extends BasePage {
       CommodityDetailsDescription(itemId) -> Detail(values(description))
     )
   }
+
+  private val chemicalCommodityCodes: Seq[String] =
+    List("28", "29", "38")
+
+  def hasChemicalCommodityCode: Boolean = {
+    val commodityDetail = detail(CommodityDetailsCode(itemId))
+    commodityDetail match {
```

Comment:
Do you really need the pattern matching? Cannot just be...
```
chemicalCommodityCodes.exists(_.startsWith(commodityDetail))
```

Reply:
done